uavcan/equipment/gnss/1063.Fix2.uavcan

```diff
@@ -0,0 +1,85 @@
+#
+# GNSS ECEF and LLA navigation solution with uncertainty.
+#
+
+uavcan.Timestamp timestamp         # Global network-synchronized time, if available, otherwise zero
+
+#
+# Time solution.
+# Time standard (GPS, UTC, TAI, etc) is defined in the field below.
+#
+uavcan.Timestamp gnss_timestamp    # GNSS timestamp, if available, otherwise zero
+
+#
+# Time standard used in the GNSS timestamp field.
+#
+uint3 GNSS_TIME_STANDARD_NONE = 0  # Time is unknown
+uint3 GNSS_TIME_STANDARD_TAI  = 1
+uint3 GNSS_TIME_STANDARD_UTC  = 2
+uint3 GNSS_TIME_STANDARD_GPS  = 3
+uint3 gnss_time_standard
+
+void13   # Reserved space
+
+#
+# If known, the number of leap seconds allows to perform conversions between some time standards.
+#
+uint8 NUM_LEAP_SECONDS_UNKNOWN = 0
+uint8 num_leap_seconds
+
+#
+# Position and velocity solution
+#
+int37 longitude_deg_1e8            # Longitude degrees multiplied by 1e8 (approx. 1 mm per LSB)
+int37 latitude_deg_1e8             # Latitude degrees multiplied by 1e8 (approx. 1 mm per LSB on equator)
+int27 height_ellipsoid_mm          # Height above ellipsoid in millimeters
+int27 height_msl_mm                # Height above mean sea level in millimeters
+
+float16[3] ned_velocity            # NED frame (north-east-down) in meters per second
+
+#
+# Fix status
+#
+uint6 sats_used
+
+uint2 STATUS_NO_FIX    = 0
+uint2 STATUS_TIME_ONLY = 1
+uint2 STATUS_2D_FIX    = 2
+uint2 STATUS_3D_FIX    = 3
+uint2 status
+
+#
+# GNSS Mode
+# 
+uint3 MODE_SINGLE      = 0
+uint3 MODE_DGPS        = 1
+uint3 MODE_RTK         = 2
+uint3 MODE_PPP         = 3
+uint3 mode
+
+#
+# GNSS Sub mode
+#
+
+uint5 SUB_MODE_DGPS_OTHER    = 0
+uint5 SUB_MODE_DGPS_SBAS     = 1
+
+uint5 SUB_MODE_RTK_FLOAT     = 0
+uint5 SUB_MODE_RTK_FIXED     = 1
+
+uint5 sub_mode
+
+#
+# Precision
+#
+float16 pdop
+
+float16[<=36] covariance    # Position and velocity covariance. Units are
```

Comment:
We need `void2` before this field to ensure byte alignment. Alternatively, we can extend `mode` and `sub_mode` by 1 bit each and move `pdop` below `covariance`.